apps/aehttp/src/sc_ws_api_jsonrpc.erl

```diff
@@ -178,13 +178,16 @@ json_rpc_error_object(invalid_pubkeys      , R) -> error_obj(3, [1003], R);
 json_rpc_error_object(call_not_found       , R) -> error_obj(3, [1004], R);
 json_rpc_error_object(contract_init_failed , R) -> error_obj(3, [1007], R);
 json_rpc_error_object(not_a_number         , R) -> error_obj(3, [1008], R);
+json_rpc_error_object(wrong_vm_version     , R) -> error_obj(3, [1010], R);
 json_rpc_error_object(participant_not_found, R) -> error_obj(3, [1011], R);
 json_rpc_error_object(not_offchain_tx      , R) -> error_obj(2, [1012], R);
 json_rpc_error_object(already_onchain      , R) -> error_obj(3, [1013], R);
 json_rpc_error_object({meta, invalid}      , R) -> error_obj(3, [1014], R);
 json_rpc_error_object(invalid_fsm_id       , R) -> error_obj(3, [1016], R);
 json_rpc_error_object(bad_signature        , R) -> error_obj(3, [1017], R);
 json_rpc_error_object(not_allowed_now      , R) -> error_obj(3, [1018], R);
+json_rpc_error_object(wrong_abi_version    , R) -> error_obj(3, [1019], R);
+json_rpc_error_object(wrong_vm_version     , R) -> error_obj(3, [1020], R);
```

Comment:
Why two clauses for `wrong_vm_version`?

Reply:
First one (with code 1010) is a bug/typo

Reply:
Addressed